lab1-review.c

```diff
@@ -0,0 +1,157 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <dirent.h>
+#include <unistd.h>
+#include <sys/stat.h>
+
+//Построчный вывод значений из аргументов командной строки
+void outputToScreen (int argc, char *argv[  ] ) {
+for(int i = 1; i < argc; i++) {
+printf ("%s\n", argv[i] );
+    }
+}
+
+//Запись символов, набираемых с клавиатуры в заданный файл
+void writeToFile(int argc, char *argv[  ] ) {
+if (argc == 1) {
+printf ("Файл записи не передан.");
+exit(-1);
+    }
+
+if (argc>2) {
+printf ("Аргументы командной строки имеют много параметров, проверьте, что только путь до файла передан.");
+exit(-1);
+    }
+
+FILE *F1 = fopen(argv[1], "w");
+if (F1 == NULL) {
+printf ("Невозможно открыть файл");
+exit(-1);
+    }
+
+printf ("Введите текст, для записи в файл. При нажатии на ESC будет произведён выход");
+int code;
+while ((code = getchar()) != 27) {
+int putCode = fputc((char)code, F1);
+if (putCode == EOF){
+printf ("Запись в файле произошла неудачно.");
+exit(-1);
+        }
+    }
+int closeCode = fclose (F1);
+if (closeCode == EOF){
+printf ("Закрытие потока работы с файлом завершилось ошибкой.");
+exit(-1);
+    }
+printf ("Запись в файл осуществлена");
+}
```

Comment:
Тоже самое, плохое форматирование
```suggestion
void writeToFile(int argc, char *argv[  ] ) {
    if (argc == 1) {
        printf ("Файл записи не передан.");
        exit(-1);
    }
    
    if (argc>2) {
        printf ("Аргументы командной строки имеют много параметров, проверьте, что только путь до файла передан.");
        exit(-1);
   }

    FILE *F1 = fopen(argv[1], "w");
    if (F1 == NULL) {
        printf ("Невозможно открыть файл");
        exit(-1);
    }

    printf ("Введите текст, для записи в файл. При нажатии на ESC будет произведён выход");
    int code;
    while ((code = getchar()) != 27) {
        int putCode = fputc((char)code, F1);
        if (putCode == EOF){
            printf ("Запись в файле произошла неудачно.");
            exit(-1);
        }
    }
   int closeCode = fclose (F1);
   if (closeCode == EOF){
       printf ("Закрытие потока работы с файлом завершилось ошибкой.");
       exit(-1);
   }
   printf ("Запись в файл осуществлена");
}
```